Source/Core/Logging/FramesLogEvent.swift

```diff
@@ -49,8 +49,8 @@ enum FramesLogEvent: Equatable, PropertyProviding {
 
     private var typeIdentifierSuffix: String {
         switch self {
-        case .checkoutAPIClientInitialised:
-            return "checkout_api_client_initialised"
+        case .paymentJourneyStart:
+            return "payment_journey_start"
```

Comment:
@deepesh-vasthimal-cko & @patrick-hoban-cko follow up on the Slack discussion, let me know if I was convincing on the need to separate this event name from syntactically getting tied to the VC init and whether you have alternative names for event.

Reply:
I'll let @deepesh-vasthimal-cko give his steer.

Reply:
@alex-ioja-yang-cko let's go with _payment_form_initialised_

That means we have more choice later on if there's another way to begin the payment journey in the future (for example, what if we have some other sort of integration option of the SDKs, I'd want to be distinct about which event I'm actually looking at).